docs/CreateEKSClusterForScalarDB.md

```diff
@@ -19,27 +25,33 @@ When deploying ScalarDB Server, you must:
 
 The following are some recommendations for deploying ScalarDB Server. These recommendations are not required, so you can choose whether or not to apply these recommendations based on your needs.
 
-### Use 4vCPU / 8GB memory nodes for the worker node in the ScalarDB Server node group
+### Create at least three worker nodes and three pods
 
-From the perspective of commercial licenses, resources for one pod running ScalarDB Server are limited to 2vCPU / 4GB memory. In addition, we recommend deploying one ScalarDB Server pod and one Envoy pod on one worker node.
+To ensure that the EKS cluster has high availability, you should use at least three worker nodes and deploy at least three pods spread across the worker nodes. You can see the [sample configurations](../conf/scalardb-custom-values.yaml) of `podAntiAffinity` for making three pods spread across the worker nodes.
 
-In other words, the following components run on one worker node:
+{% capture notice--info %}
+**Note**
 
-* ScalarDB Server pod (2vCPU / 4GB)
-* Envoy proxy
-* Kubernetes components
+If you place the worker nodes in different [availability zones](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html) (AZs), you can withstand an AZ failure.
+{% endcapture %}
 
-With this in mind, you should use a worker node that has 4vCPU / 8GB memory resources. We recommend running only the above components on the worker node for ScalarDB Server. However, if you want to run other pods on the worker node for ScalarDB Server, you should use a worker node that has more than 4vCPU / 8GB memory.
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+### Use 4vCPU / 8GB memory nodes for the worker node in the ScalarDB Server node group
 
-Note that you should configure resource limits based on your system's workload if the Envoy pod exceeds the above resource usage. In addition, you should consider scaling out the worker node and the ScalarDB Server pod if the ScalarDB Server pod exceeds the above resource usage and if latency is high (throughput is low) in your system.
+From the perspective of commercial licenses, resources for one pod running ScalarDB Server are limited to 2vCPU / 4GB memory. In addition, some pods other than ScalarDB Server pods could exist on the worker nodes.
 
-### Create a node group for other application pods than ScalarDB Server pods
+In other words, the following components could run on one worker node:
```

Comment:
Does the wording for this depend on how we word a similar sentence in a different PR?: [related comment](https://github.com/scalar-labs/scalar-kubernetes/pull/213/files#r1382880277)